### DISCUSSION TITLE
Export issue with PRO version on Mac

### DISCUSSION THREAD

**Discussion by mrmangum** [nesbox/TIC-80#1432] [Q&A]
I am trying to export native so as to upload my WIP cart to my tic80.com profile.
The file size is 3.7mb and I keep getting "413 Request Entity Too Large" error on the tic80.com website.

Why would my file size be so large when it is only 159 lines of code, no sprites and no sfx?

**Comment by nesbox** [comment]
Hi, you can upload only `.tic` files to the https://tic80.com website.
Use `folder` command to find your cart.
Thanks.

**Reply by mrmangum** [reply]
Thank you. I didn't realize `export native` was creating a different kind of file.
But now it won't upload because I need a cover. For some reason, f7 key won't work to generate cover.
Any ideas? f7 works in my other apps.

**Reply by mrmangum** [reply]
NVM I didn't realize I needed to be running the game to generate the cover.
Sorry to bug you, but thank you for all your help!
